src/Controller/BlockAdminController.php

```diff
@@ -49,19 +47,15 @@ public function savePositionAction(Request $request): Response
         $this->admin->checkAccess('savePosition');
 
         try {
-            $params = $request->get('disposition');
+            // TODO: Change to $request->query->all('filter') when support for Symfony < 5.1 is dropped.
+            $params = $request->request->all()['disposition'] ?? [];
 
-            if (!\is_array($params)) {
+            if ([] === $params) {
                 throw new HttpException(400, 'wrong parameters');
             }
 
-            $result = $this->container->get('sonata.page.block_interactor')->saveBlocksPosition($params, false);
-
+            $result = $this->container->get('sonata.page.block_interactor')->saveBlocksPosition($params);
             $status = 200;
-
-            $pageAdmin = $this->container->get('sonata.page.admin.page');
-            $pageAdmin->setRequest($request);
-            $pageAdmin->update($pageAdmin->getSubject());
```

Comment:
If I don't remove this, this method only make sense in the context of child, but if I remove it, the page itself won't get the edited set to true.

I guess I will have to recover those lines, but that means I have to limit the usage of this route to child of blocks, do you agree @VincentLanglet ?

Otherwise we could move the setEdited(true) to the prePersist of block admin (it is now on the blockAdmin, but that does not get triggered when saving through modelManager).

What to do, @VincentLanglet . I like to remove those 3 lines here, and add on prePersist and preUpdate, will try that way, and wait for your feedback.